examples/ipython/Old Format.ipynb

```diff
@@ -1009,9 +1009,9 @@
       "grad<b = (-D{y}b__xy - D{z}b__xz)*e_x + (D{x}b__xy - D{z}b__yz)*e_y + (D{x}b__xz + D{y}b__yz)*e_z\n",
       "grad>b =  0 \n",
       "\n",
-      "s*grad = \\boldsymbol{e}_{x}*s {\\left (x,y,z \\right )} \\frac{\\partial}{\\partial x} + \\boldsymbol{e}_{y}*s {\\left (x,y,z \\right )} \\frac{\\partial}{\\partial y} + \\boldsymbol{e}_{z}*s {\\left (x,y,z \\right )} \\frac{\\partial}{\\partial z}\n",
-      "s^grad = \\boldsymbol{e}_{x}*s {\\left (x,y,z \\right )} \\frac{\\partial}{\\partial x} + \\boldsymbol{e}_{y}*s {\\left (x,y,z \\right )} \\frac{\\partial}{\\partial y} + \\boldsymbol{e}_{z}*s {\\left (x,y,z \\right )} \\frac{\\partial}{\\partial z}\n",
-      "s<grad = \\boldsymbol{e}_{x}*s {\\left (x,y,z \\right )} \\frac{\\partial}{\\partial x} + \\boldsymbol{e}_{y}*s {\\left (x,y,z \\right )} \\frac{\\partial}{\\partial y} + \\boldsymbol{e}_{z}*s {\\left (x,y,z \\right )} \\frac{\\partial}{\\partial z}\n",
+      "s*grad = e_x*s*D{x} + e_y*s*D{y} + e_z*s*D{z}\n",
+      "s^grad = e_x*s*D{x} + e_y*s*D{y} + e_z*s*D{z}\n",
+      "s<grad = e_x*s*D{x} + e_y*s*D{y} + e_z*s*D{z}\n",
```

Comment:
These are now way more pleasant to read in a terminal, and way mroe consistent with the lines immediately above

Reply:
These are completely breaking changes, changing the behavior from printing LaTeX to terminal style output.....

Reply:
Yes, but this in a test that is not supposed to print latex!

Reply:
(you can tell because `s^grad` is not valid latex anyway, nor is the line above, `grad<b = (-D{y}b__xy - D{z}b__xz)*e_x + (D{x}b__xy - D{z}b__yz)*e_y + (D{x}b__xz + D{y}b__yz)*e_z`)